auth/tokens.go

```diff
@@ -30,28 +30,54 @@ type TokenData interface {
 }
 
 type SimpleTokenData struct {
-	isAdmin bool
-	Name    string `json:"name"`
-	Desc    string `json:"desc"`
-	Lang    string `json:"lang"`
-	Token   string `json:"token"`
-	id      int
-	Expiry  time.Time `json:"expiry,omitempty"`
+	Name       string         `json:"name"`
+	Desc       string         `json:"desc"`
+	Lang       string         `json:"lang"`
+	Token      string         `json:"token"`
+	Expiry     time.Time      `json:"expiry,omitempty"`
+	Extensions map[string]any `json:"extensions,omitempty"`
+
+	Id    int  `json:"id"`
+	Admin bool `json:"admin"`
```

Comment:
I don't like changing name f property isAdmin. This make code not clean  Also I don't understand why need to export this property ? func (s *SimpleTokenData) IsAdmin() is for this

Reply:
I cannot leave it IsAdmin as long as we have a method with the same name and it is not allowed. And we've talked about it. I need this exported, because as long as the field is unexported it is lost in marshaling/unmarshaling and saving to some json field for example. You said you agree to make it exported. And now you say you don't....

Reply:
Remove method?